services/blockchain-coordinator/shared/scheduler.js

```diff
@@ -127,6 +129,44 @@ const initIndexingScheduler = async () => {
 	// }
 };
 
+const scheduleMissingBlocksIndexing = async (params = {}) => {
```

Comment:
Please consider splitting the logic in this method into two methods: `getAllMissingBlocks` and `scheduleBlocksIndexing` that ensure separation of concerns.